src/compositions/Checklist/Checklist.stories.tsx

```diff
@@ -0,0 +1,56 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Checkbox, CheckboxState, CheckboxStyle } from "@elements/Checkbox/Checkbox";
+import { Meta, Story } from "@storybook/react";
+import { useState } from "react";
+import { Checklist, ChecklistProps, Direction } from "./Checklist";
+
+export default {
+    title: "Components/Checklist",
+    component: Checklist,
+    argTypes: {
+        direction: {
+            options: [Direction.Horizontal, Direction.Vertical],
+            control: { type: "radio" },
+        },
+    },
+} as Meta<ChecklistProps>;
+
+const Template: Story<ChecklistProps> = (args) => {
+    const [checked, setChecked] = useState<CheckboxState[]>([
+        CheckboxState.Unchecked,
+        CheckboxState.Checked,
+        CheckboxState.Mixed,
+    ]);
+
+    const updateCheckState = (index: number) => (isChecked: boolean) => {
+        checked[index] = isChecked ? CheckboxState.Checked : CheckboxState.Unchecked;
+        setChecked([...checked]);
+    };
+
+    return (
+        <Checklist {...args}>
+            <Checkbox
+                checked={checked[0]}
+                onChange={updateCheckState(0)}
+                style={CheckboxStyle.Primary}
+                label="Checkbox label"
+                tooltip="Random Tooltip"
+            />
+            <Checkbox
+                checked={checked[1]}
+                onChange={updateCheckState(1)}
+                label="Checkbox label"
+                note="Note about this input"
+                disabled
+            />
+            <Checkbox checked={checked[2]} onChange={updateCheckState(2)} label="Checkbox label" />
+        </Checklist>
+    );
+};
+
+export const Default = Template.bind({});
```

Comment:
If there's only one story I'd export it directly so there's no sublabel in storybook

Reply:
bc03773